install/infra/terraform/k3s/variables.auto.tfvars

```diff
@@ -0,0 +1,5 @@
+kubeconfig = "./kubeconfig"
+
+gcp_project = "sh-automated-tests"
+gcp_region  = "europe-west1"
+gcp_zone    = "europe-west1-b"
```

Comment:
Is this file helpful? I'm asking because `variables.tf` already has defaults and for the sake of simplicity it would be nice to keep the number if files low.